lib/system.nim

```diff
@@ -3126,3 +3126,9 @@ export io
 
 when not defined(createNimHcr) and not defined(nimscript):
   include nimhcr
+
+when notJSnotNims and not defined(nimSeqsV2):
+  proc prepareStrMutation*(s: var string) {.inline.} =
+    ## String literals(i.e. "abc", etc) in ARC/ORC mode are "copy on write",
```

Comment:
```suggestion
    ## String literals(e.g. "abc", etc) in the ARC/ORC mode are "copy on write",
```

Reply:
While this change is fine (most people will understand what it means anyway), `i.e.` is correct here as well - https://en.wiktionary.org/wiki/i.e. means "that is", so the sentence reads like "String literals (that is - "abc", etc)".